### PR TITLE
Fix Guide Sidebar and Titles

### PR DESCRIPTION
**Pull request by turnipdabeets**
Some integrations were missing on the sidebar and updated titles

Screenshot 2023-01-05 at 12 09 03 PM

### CODE REVIEW

**Comment by Will5** [comment]
This makes sense to me. I also notice the sidebars between "Documentation" and "Guides" show different SSOs. Was this intended (CC: @nmelo)? I think "Documentation" > "SSO Integration Guides", "Guides" > "SSO Integrations" and "Guides" > "Integration with other SSOs" should be the same. Is there an easy way to synchronize? 

Screenshot 2023-01-05 at 1 23 08 PM

Screenshot 2023-01-05 at 1 23 20 PM

**Comment by turnipdabeets** [comment]
> This makes sense to me. I also notice the sidebars between "Documentation" and "Guides" show different SSOs. Was this intended (CC: @nmelo)? I think "Documentation" > "SSO Integration Guides", "Guides" > "SSO Integrations" and "Guides" > "Integration with other SSOs" should be the same. Is there an easy way to synchronize?
> 
> Screenshot 2023-01-05 at 1 23 08 PM
> 
> Screenshot 2023-01-05 at 1 23 20 PM

Good catch, I updated this too!

File: sidebars.js [hunk 1]
```diff
@@ -148,6 +148,26 @@ const sidebars = {
           label: 'Integrate with Curity ⧉',
           href: '/guides/sso-integrations/integrate-with-curity',
         },
+        {
+          type: 'link',
+          label: 'Integrate with Curity ⧉',
+          href: '/guides/sso-integrations/integrate-with-curity',
+        },
```
Comment (Will5) [inline sidebars.js:155]: Curity is twice as important 😆

Comment (turnipdabeets) [reply sidebars.js]: Good catch! removed the duplicate

**Review by Will5 — APPROVED** [review]

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**